### PR TITLE
One create footer

### PR DESCRIPTION
**Pull request by ghost**
Does it look ok?

### CODE REVIEW

**Comment by SebastianGerS** [comment]
@X3n0m0rphose 
Looks good, hade to change some things to look at it tought.

* If you look in the web file the link in there is to a layouts.heeder file if you want to look at your layouts file it should link to layouts.footer also the footer.blade file should be in the layouts folder (for this particular path to work).

also try to make the icons a bit bigger, and add a icon for mail like in the prototype: 
https://projects.invisionapp.com/share/N8EWQRNK3#/screens/268772024

also if we are to follow the prototype the icons for mail and location shuold be to the left, see if you can try to inplement that

**Comment by ghost** [comment]
Thank you so much for feedback, just got a little confuse there in the end.
I'll fix that

Den 14 dec. 2017 14:59 skrev "SebastianGerS" <notifications@github.com>:

**Comment by SebastianGerS** [comment]
@X3n0m0rphose looks good, seams to work both for smal screens and desktops, the content is still centered as far as i can tell, but meaby thats fine for now, we will fix it leater, anna is working a new design so the footer might change anyway

what do the rest of the group think? 
@AnnaRoss  @Tokidaze @Laloo89  @Oleg-Lopes

**Comment by ghost** [comment]
Yes i will fix the text

Den 15 dec. 2017 12:46 skrev "SebastianGerS" <notifications@github.com>:

File: resources/views/movies/footer.blade.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (AnnaRoss) [inline resources/views/movies/footer.blade.php]: Please remove all of the markup above the `<footer>`-tag

File: resources/views/movies/footer.blade.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+<footer class="footer">
+
```
Comment (AnnaRoss) [inline resources/views/movies/footer.blade.php]: Remove this empty line, please :smile_cat:

File: resources/views/movies/footer.blade.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+@include('includes.footer')
+<div class="container">
+  <div class="content has-text-centered">
+  <p><i class="fa fa-map-marker" aria-hidden="true">Location</i><p>
```
Comment (AnnaRoss) [inline resources/views/movies/footer.blade.php]: Indentention needed

File: resources/views/movies/footer.blade.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<footer class="footer">
+
+@include('includes.footer')
+<div class="container">
+  <div class="content has-text-centered">
+  <p><i class="fa fa-map-marker" aria-hidden="true">Location</i><p> 
+  <i class="fa fa-twitter" aria-hidden="true"></i>
+  <i class="fa fa-facebook-official" aria-hidden="true"></i>
+
+<i class="fa fa-github" aria-hidden="true"></i>
```
Comment (AnnaRoss) [inline resources/views/movies/footer.blade.php]: Indentention needed

File: routes/web.php [hunk 5]
```diff
@@ -15,6 +15,10 @@
     return view('welcome');
 });
 
+Route::get('/footer', function () {
```
Comment (AnnaRoss) [inline routes/web.php:18]: This should be removed before staging.

**Comment by Laloo89** [comment]
Removed markup above <footer>
Changed background to match new prototype

**Comment by ghost** [comment]
Gr8!

Den 15 dec. 2017 14:48 skrev "Laila H." <notifications@github.com>:

Removed markup above

Changed background to match new prototype

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/chas-academy/berserkers-06-imdb-clone/pull/50#issuecomment-352009938>,
or mute the thread

**Comment by Laloo89** [comment]
Updated footer again to match new design,
there are still a few things that can be way better, but thought that Hossein could fix that,
since its the same look/code for the header.

**Pull request closed without merging**